frontend/src/pages/LogCode.jsx

```diff
@@ -0,0 +1,112 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { TextField, InputAdornment } from "@mui/material";
+import { Article } from "@mui/icons-material";
+import Logo from "../assets/images/logo.png";
+import "./styles/LogCode.css";
+
+const LogCode = () => {
+  const [setLogbookCode] = useState("");
+  const [termsAccepted, setTermsAccepted] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const navigate = useNavigate();
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    if (!termsAccepted) {
+      return alert("Please accept the terms and conditions");
+    }
+    try {
+      setLoading(true);
+      await new Promise((resolve) => setTimeout(resolve, 1000));
+      navigate("/");
+    } catch {
+      alert("Something went wrong. Please try again.");
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  return (
+    <div className="logcode-page">
+      <div className="container">
+        <div className="form-container">
+          <form onSubmit={handleSubmit}>
+            <h1>Logbook code entry</h1>
+            <h2 className="form-description">
+              Please enter in your logbook code in order to finish registering
+              your account.
+            </h2>
+
+            <div className="code-group">
+              <label htmlFor="logbook-code">Logbook code</label>
+              <TextField
+                id="logbook-code"
+                type="text"
+                variant="outlined"
+                fullWidth
+                placeholder="Enter logbook code"
+                onChange={(e) => setLogbookCode(e.target.value)}
+                InputProps={{
+                  startAdornment: (
+                    <InputAdornment position="start">
+                      <Article />
+                    </InputAdornment>
+                  ),
+                }}
+                required
+              />
+            </div>
+
+            <div className="help-text">
+              <a
+                href="#"
+                onClick={(e) => {
+                  e.preventDefault();
+                  alert("Located inside your logbook.");
```

Comment:
Instead of making this an alert, could we perhaps make this a tooltip, similar to this?
![image](https://github.com/user-attachments/assets/c0d325d2-912f-4157-b205-6037d8ebe94f)